ddtrace/appsec/iast/__init__.py

```diff
@@ -28,6 +28,8 @@ def wrapped_function(wrapped, instance, args, kwargs):
     )
     return wrapped(*args, **kwargs)
 """  # noqa: RST201, RST213, RST210
+# This import will trigger the change from regular loader to IAST loader if required
```

Comment:
Can we avoid doing import behaviour? Having logic run at import time is implicit and quite error prone. Could we instead have a function that is called at an explicit point in code?

Reply:
Sure, I can move it to ``ddtrace.bootstrap.sitecustomize`` for example